src/annotator/config/index.js

```diff
@@ -162,8 +198,33 @@ export function getConfig(appContext = 'annotator', window_ = window) {
   let filteredKeys = configurationKeys(appContext);
   filteredKeys.forEach(name => {
     const configDef = configDefinitions[name];
+    const hasDefault = configDef.defaultValue !== undefined; // A default could be null
+    const isURLFromBrowserExtension = isBrowserExtension(
+      urlFromLinkTag(window_, 'sidebar', 'html')
```

Comment:
When initially reading this I wondered what would happen if the link tag was missing. This triggers an exception and the client will fail to boot.

Reply:
Sure, but that was always the case, or are you saying this is a difference with this PR?

Reply:
Oh, I didn't mean there was a change in this PR. I'm just making a note of a query that popped into my head while reading this.